src/components/shared/AutoComplete/renderInput.tsx

```diff
@@ -1,9 +1,48 @@
-import React, { ReactElement } from 'react';
+import React, { ReactElement, useEffect, useState } from 'react';
 
+import { Animated } from 'react-native';
 import { Input } from './styles';
 
 const RenderInput = (props): ReactElement => {
-  return <Input {...props} />;
+  const [animatedIsFocused] = useState(new Animated.Value(props.on ? 1 : 0));
+  useEffect(() => {
+    Animated.timing(animatedIsFocused, {
+      toValue: props.on ? 1 : 0,
+      duration: 200,
+    }).start();
```

Comment:
can you try give some delay with [`Animated.delay()`](https://facebook.github.io/react-native/docs/animated#delay) so that a user can see the animations in the renderInput more clearly? Currently it's hard to recognize (especially when the options list is opened) because the renderInput is moving rapidly to the top of the screen.